android/app/src/main/res/values-night/styles.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <!-- Theme applied to the Android Window while the process is starting when the OS's Dark Mode setting is on -->
-    <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
-        <!-- Show a splash screen on the activity. Automatically removed when
-             the Flutter engine draws its first frame -->
-        <item name="android:windowBackground">@drawable/launch_background</item>
-        <item name="android:forceDarkAllowed">false</item>
-        <item name="android:windowFullscreen">false</item>
-        <item name="android:windowDrawsSystemBarBackgrounds">false</item>
-        <item name="android:windowLayoutInDisplayCutoutMode">shortEdges</item>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Shim in the system UI overrides -->
+    <style name="SplashTheme.NightAdjusted" parent="Theme.SplashScreen">
+        <item name="android:windowLightStatusBar">false</item>
+        <item name="android:windowLightNavigationBar" tools:targetApi="27">false</item>
     </style>
+
     <!-- Theme applied to the Android Window as soon as the process has started.
          This theme determines the color of the Android Window while your
          Flutter UI initializes, as well as behind your Flutter UI while its
          running.
 
          This Theme is only used starting with V2 of Flutter's Android embedding. -->
     <style name="NormalTheme" parent="@android:style/Theme.Black.NoTitleBar">
-        <item name="android:windowBackground">@color/launch_black</item>
+        <item name="android:windowBackground">?android:colorBackground</item>
```

Comment:
`?android:colorBackground` is what was already being used on API 31+, so I used it here as well.

Reply:
IIRC I put this because I'd rather have an always black screen than a white one while the system itself is in dark mode.

Doest the `?android:colorBackground` follow system dark mode?

Reply:
It should, since it just references a themed value and the current theme is overridden to dark when the device is in dark mode. I'm not completely sure it's necessary (or if it's the desired theme attr to use), but I didn't want to change anything that could potentially introduce a regression. It may be better to just hardcode to black/white if that's the desired behavior (I believe that's what ?colorBackground resolves to anyway).

Reply:
I posted in my other comment, but `?android:colorBackground` is coming from `flutter_native_splash`.